Note that findByUid() and findByIdentifier() are not affected by the … (#5556)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/Repository/Index.rst

    The "magic" find methods `findByX()`, `findOneByX()` and `countByX()` have
    been removed. See :ref:`t3coreapi/13:extbase-repository-find-by-magic-migration`

>     The (not-magic) methods `findByUid()` and `findByIdentifier()` did **not**

Reviewer: ```suggestion
    The (not-magic) methods `findByUid()` and `findByIdentifier()` have **not**
```

>     get deprecated or removed, and are still valid to be used.

Reviewer: ```suggestion
    been deprecated or removed, and are still valid.
```
